<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloudMusicWindow</class>
 <widget class="QMainWindow" name="CloudMusicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CloudMusicWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
background-image: url(:/images/background.png);}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1,2,1,1">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="2,1,2,1,4,1,2,1,2">
        <item row="2" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/images/cycle-play.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="6" rowspan="2">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" rowspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="5">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="7">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="6">
         <widget class="QPushButton" name="pushButton_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/images/play-next.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="8">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="8">
         <widget class="QPushButton" name="musicLyricTableButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-image: url(:/images/default-music-list.png);
}
QPushButton[displayMode=&quot;musicTableMode&quot;]{
	border-image: url(:/images/default-music-list.png);
}
QPushButton[displayMode=&quot;musicLyricMode&quot;]{
	border-image: url(:/images/music-list.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="8">
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4" rowspan="4">
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/images/pause.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/images/play-pri.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,8,1,1">
        <item>
         <widget class="QLabel" name="playTimeLabel">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>00:00</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="musicPlaySlider">
          <property name="styleSheet">
           <string notr="true">QSlider
{
    border-color: #bcbcbc;
}

QSlider::groove:horizontal
{                                
     border: 1px solid #999999;                             
     height: 3px;                                           
     margin: 0px 0;                                         
     left: 5px; right: 5px; 
 }
 
QSlider::handle:horizontal
{                               
     border: 0px ;                           
     border-image: url(:/images/slider-handle.png);
     width: 15px;                                           
     margin: -7px -7px -7px -6px;                  
} 

QSlider::add-page:horizontal
{
    background:white;
}

QSlider::sub-page:horizontal
{                               
     background:red;                
}
</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="musicDurationLabel">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>04:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,10">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLineEdit" name="searchLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit 
{
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: yellow;
    selection-background-color: darkgray;
 }
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="styleSheet">
           <string notr="true">/* The tab widget frame */
QTabWidget::pane 
{ 
    border-top: 2px solid #55aaff;
    position: absolute;
    top: -0.5em;
 }

 QTabWidget::tab-bar
 {
     alignment: center;
 }

  /* Style the tab using the tab sub-control. Note that it reads
   QTabBar_not_ QTabWidget */
  QTabBar::tab
  {    
      background-color: rgb(85, 170, 255);
      border: 2px solid #00ff7f;
      border-bottom-color: #C2C7CB; /* same as the pane color */
      border-top-left-radius: 4px;
      border-top-right-radius: 4px;
      min-width: 8ex;
      padding: 2px;
  }

  QTabBar::tab:hover
  {  
      background-color: rgb(255, 0, 255);
  }

  QTabBar::tab:selected
  {            
     background-color: rgb(255, 0, 127);
     /* same as pane color */
     border-bottom-color: rgb(85, 170, 255);
  }
</string>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>本地歌曲</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableWidget" name="localMusicTable">
              <property name="styleSheet">
               <string notr="true">QTableWidget
{
    /*前景色：文字颜色*/    
    color: rgb(0, 0, 0);
   
    /*表格中的网格线条颜色*/    
    /*gridline-color:rgb(255, 224, 205);*/
    
    background-color: rgb(200, 200, 0);
    background-color: rgb(255, 170, 0);
        
    /*设置交替颜色，需要在函数属性中设置:
    tableWidget-&gt;setAlternatingRowColors(true)
    */
    alternate-background-color: rgb(85, 85, 255);
    selection-color:red;    /*鼠标选中时前景色：文字颜色*/
    selection-background-color:yellow;   /*鼠标选中时背景色*/
    border:1px solid gray;  /*边框线的宽度、颜色*/
    border:0px;    /*去除边界线*/
    /*border-radius:5px;*/
    /*padding:10px 10px;*/  /*表格与边框的间距*/
}

QHeaderView::section
{               
     background-color: rgb(85, 255, 0);
     font-weight  : bold;    
     color: rgb(255, 0, 0);
     border: 0px solid;
}

QTableWidget::Item
{
    border:0px solid rgb(255,0,0);
    border-bottom:1px solid rgb(255,0,0);
}
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>网络歌曲</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableWidget" name="networkMusicTable">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QTableWidget
{
    /*前景色：文字颜色*/    
    color: rgb(0, 0, 0);
   
    /*表格中的网格线条颜色*/    
    /*gridline-color:rgb(255, 224, 205);*/
    
    background-color: rgb(200, 200, 0);
    background-color: rgb(255, 170, 0);
        
    /*设置交替颜色，需要在函数属性中设置:
    tableWidget-&gt;setAlternatingRowColors(true)
    */
    alternate-background-color: rgb(85, 85, 255);
    selection-color:red;    /*鼠标选中时前景色：文字颜色*/
    selection-background-color:yellow;   /*鼠标选中时背景色*/
    border:1px solid gray;  /*边框线的宽度、颜色*/
    border:0px;    /*去除边界线*/
    /*border-radius:5px;*/
    /*padding:10px 10px;*/  /*表格与边框的间距*/
}

QHeaderView::section
{               
     background-color: rgb(85, 255, 0);
     font-weight  : bold;    
     color: rgb(255, 0, 0);
     border: 0px solid;
}

QTableWidget::Item
{
    border:0px solid rgb(255,0,0);
    border-bottom:1px solid rgb(255,0,0);
}
QTableWidget::Item:selected
{
	color: rgb(0, 255, 255);
    background-color:rgb(0, 0, 127)
}

</string>
              </property>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>mp3Url</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>lyricUrl</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>albumUrl</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>歌曲名</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>专辑名</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>时长</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="lyricTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
